components/data.ts

```diff
@@ -95,6 +95,7 @@ export const MainRoutes = (): Link[] => [
       { title: t('Web_polyfill_CDN'), path: 'https://polyfill.kaiyuanshe.cn/' },
       { title: t('open_source_mirror'), path: 'http://mirror.kaiyuanshe.cn/' },
       { title: t('license_tool'), path: '/tool/license-filter' },
+      { title: t('activity_articles_calendar'), path: '/calendar' },
```

Comment:
```suggestion
      { title: t('activity_articles_calendar'), path: '/activity/calendar' },
```
我觉得把它作为“精彩活动”的一个子菜单“活动日历”比较好，原有的链接变为“主办活动”子菜单。